<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogTrans</class>
 <widget class="QDialog" name="dialogTrans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transition</string>
  </property>
  <widget class="QPushButton" name="buttonOKTrans">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditGuard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditExpr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGuard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Guards</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelToken">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Expression</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
